src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -123,8 +123,8 @@ public void Crossgen(string outputPath)
             {
                 foreach (var asset in package.Assets)
                 {
-                    var subDir = CreateSubDirectory(outputPath, package, asset);
-                    var targetPath = Path.Combine(subDir.FullName, asset.FileName);
```

Comment:
Ok, we leave only `sha512` and `dll` file in the packages cache. So in this case, can I simply the rules to be: lower case for all file path except the file name of the assemblies (dll), which are kept as they were.